dds_web/templates/user/password_reset_completed.html

```diff
@@ -15,21 +15,25 @@ <h1>Password updated!</h1>
     </div>
     {% endif %}
     {% endwith %}
-    <p><b>You have lost access to your active projects when you reset your password.</b></p>
-    <p>Please contact the units where you have active projects to restore your access.</p>
 
-    <p>You have active projects in<p>
-    <ul>
-      {% for unit_name, contact in units_to_contact.items()%}
-        <li>{{ unit_name }}: {{ contact }}</li>
-      {% endfor %}
-    </ul>
+    {% if units_to_contact %}
+    <p><b>You have lost access to your active projects when you reset your
+            password.</b></p>
+    <p>Please contact the units where you have active projects to restore your
+        access.</p>
 
-    <small class="text-muted ml-2">
-        <a href="{{ url_for('auth_blueprint.login')
-            }}">
-            Back to Login
-        </a>
-    </small>
+    <p>You have active projects in<p>
+            <ul>
+                {% for unit_name, contact in units_to_contact.items()%}
+                <li>{{ unit_name }}: {{ contact }}</li>
+                {% endfor %}
+            </ul>
+            {% endif %}
+            <small class="text-muted ml-2">
+                <a href="{{ url_for('auth_blueprint.login')
+                    }}">
+                    Back to Login
+                </a>
+            </small>
 
-{% endblock %}
+            {% endblock %}
```

Comment:
Why is endblock indented so much?` {% block content %}` is not indented at all.

Reply:
I can't even change it, cause the save just does it again

Reply:
Seems to be a common problem in vscode but the proposed fix is not applicable for me